### ISSUE TITLE
My screenshots upload with errors

### ISSUE THREAD

**Issue by tyronx** [geospiza-fortis/owlrepo-client#42]
"Processed 3/3 with 3 errors."

Files attached, they are unedited. 

![MapleLegends 29-04-2022 17-34-46](https://user-images.githubusercontent.com/762006/165967361-ff5bbd78-7172-4980-99b4-4a759514b0cc.png)
![MapleLegends 29-04-2022 17-34-45](https://user-images.githubusercontent.com/762006/165967383-b4a91098-cc88-48bb-b581-dcc3cd1b5b63.png)
![MapleLegends 29-04-2022 17-34-44](https://user-images.githubusercontent.com/762006/165967399-4cdca37d-1464-4d1f-95b7-1df74768be31.png)

**Comment by tyronx** [comment]
Maybe its because I play on a 4k screen?

**Comment by tyronx** [comment]
Nope, its because i made those screenshots in fullscreen mode. Outside fullscreen it works fine. Might wanna add a line about that in the upload instructions

**Comment by geospiza-fortis** [comment]
Oh? That's actually interesting. I've been doing some active development so maybe I can take a look into this sooner than later. I appreciate the bug report.

**Comment by abreathingcorpse** [comment]
I was wondering why it didn't work for me as well. Guess I know now.
